app/src/main/res/anim/slide_in_left.xml

```diff
@@ -18,5 +18,5 @@
 <set xmlns:android="http://schemas.android.com/apk/res/android">
     <translate android:fromXDelta="-100%" android:toXDelta="0%"
         android:fromYDelta="0%" android:toYDelta="0%"
-        android:duration="700"/>
+        android:duration="400"/>
```

Comment:
Would you mind define the `duration` in a resource file and people can change it just in one place later?